### DISCUSSION TITLE
Current Char Scale Detection Trigger

### DISCUSSION THREAD

**Discussion by shinra358** [ikemen-engine/Ikemen-GO#1978] [Feature Request]
### Is your feature request related to a problem? Please describe.

There's no way to tell what scale you are currently in to be able to trigger things when you are at a certain scale.

### Describe the solution you'd like

If stagezoom makes your char go from a scale of 1, 1 to a scale of 0.75, 0.75 for example, we can use this trigger to say if current character scale is 0.75, 0.75 then trigger this sctrl:

CurScale X = 0.75 || CurScale Y = 0.75

then we can create a Zoom or AngleDraw formula that can do things like prevent char scaling despite the current stage zoom. Useful for effects and intricate UIs.

**Comment by potsmugen** [comment]
There's the `CameraZoom` trigger from Mugen 1.1.

**Comment by shinra358** [comment]
oh snap, i didnt see that. I think that will do the trick! I have to experiment. Thanks!

**Reply by shinra358** [reply]
yes it works as i wanted, i figured out the formula that allows for what i wanted to do in the initial post.
